src/pages/preferences/PreferencesTabs.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { Skeleton } from "antd";
+import { H1 } from "components/Typography/index";
+import { ProfileTab } from "./ProfileTab";
+import { UserSettings } from "gql/generated/types";
+interface PreferenceTabsProps {
+  tabKey: string;
+  userSettings: UserSettings;
+  loading: boolean;
+  error: any;
+}
+export const PreferencesTabs: React.FC<PreferenceTabsProps> = ({
+  tabKey,
+  userSettings,
+  loading,
+  error,
+}) => {
+  return (
+    <Container>
+      <H1>{mapUrlTabToTitle[tabKey]}</H1>
+      {loading && <Skeleton active={true} />}
+      {!loading && (
+        <MapUrlTabToPage tabKey={tabKey} userSettings={userSettings} />
+      )}
+    </Container>
+  );
+};
+
+const MapUrlTabToPage: React.FC<{
```

Comment:
this could be easier to understand if we saved the componentMap outside of our components and included these two lines 
` const Component = componentMap[tabKey];
...
 <Component {...userSettings} />;` 
inside PreferencesTab

Reply:
Thanks. I was originally intending to do something else with this but changed directions forgot to clean it up.